go-apps/meep-virt-engine/helm/install.go

```diff
@@ -61,11 +62,14 @@ func install(chart Chart) error {
 	log.Debug("Installing chart: " + chart.ReleaseName)
 	var cmd *exec.Cmd
 	if strings.Trim(chart.ValuesFile, " ") == "" {
+		codecovLocation := strings.TrimSpace(os.Getenv("MEEP_CODECOV_LOCATION")) + chart.ReleaseName
 		cmd = exec.Command("helm", "install", chart.ReleaseName,
 			"--namespace", chart.Namespace, "--create-namespace",
 			"--set", "nameOverride="+chart.Name,
 			"--set", "fullnameOverride="+chart.Name,
-			chart.Location, "--replace", "--disable-openapi-validation")
+			chart.Location, "--replace", "--disable-openapi-validation",
+			"--set", "codecov.enabled="+strings.TrimSpace(os.Getenv("MEEP_CODECOV")),
```

Comment:
To confirm, when this flag is set on _meep-virt-engine_ it will enable codecov on all sandbox pods, even if they were not built in code codecov mode. This seems acceptable... i.e. a user wanting to enable codecov should know to build & deploy using codecov flag.